RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -611,19 +614,21 @@ namespace mkfit {
                       "h_x/F:h_y/F:h_r/F:h_phi/F:h_z/F:"
                       "h_ex/F:h_ey/F:h_er/F:h_ephi/F:h_ez/F:"
                       "ht_dxy/F:ht_dz/F:ht_dphi/F:"
-                      "h_chi2/F"
+                      "h_chi2/F:bkwd/I"
                       "\n");
                   first = false;
                 }
 
-                if (!(std::isnan(phi)) && !(std::isnan(getEta(m_Par[iI].At(itrack, 5, 0))))) {
+		if (1 && !(std::isnan(phi)) && !(std::isnan(getEta(m_Par[iI].At(itrack, 5, 0)))) && m_Label(itrack, 0, 0)>=0 && m_Label(itrack, 0, 0)==mchid && m_Label(itrack, 0, 0)==thisseedmcid )  {
```

Comment:
is `1 &&` needed?

Reply:
of course not

I need to put some other nan protections and the hit-track selection under ifdef